biogeophys/FatesPlantHydraulicsMod.F90

```diff
@@ -1654,6 +1867,101 @@ subroutine ConstrainRecruitNumber(csite,ccohort, bc_in)
     ccohort%n = min (ccohort%n, nmin) 
 
   end subroutine ConstrainRecruitNumber
+#endif
+
+  !=====================================================================================
+  subroutine ConstrainRecruitNumber(csite,ccohort, bc_in)
```

Comment:
@pnlfang did you change the algorithm or fix bugs for ConstrainRecruitNumber()?

Reply:
Ryan, it was your fix. See #721

Reply:
ok, was wondering why it is showing up as a diff, but no problem